maxq10xx/README.md

```diff
@@ -146,3 +146,12 @@ you fa shizzle!" and then terminate.
 The client will print the message it received from the server along with more
 debug and status logging and then terminate.
 
+## LWIP Example
+
+You will also find `wolfssl-lwip-client.c`.
+
+This example shows how to write a simple TLS client that uses the features
+of the Analog Devices MAXQ1065 and 1080 USS. Note that this is not a
+stand-alone application. This is part of an example that can be found in the
```

Comment:
```
...part of an example that can be found in the SDK supplied by Analog Devices.
```

Is this example our IP? Written by us. Please add how someone would go about compiling the example. i.e. if should download the SDK then add wolfssl-lwip-client.c to a specific location and build.